libmscore/property.cpp

```diff
@@ -160,32 +165,32 @@ static const PropertyData propertyList[] = {
       { P_ID::REPEAT_MEASURE,      true,  0,               P_TYPE::BOOL    },
       { P_ID::REPEAT_JUMP,         true,  0,               P_TYPE::BOOL    },
 
-      { P_ID::MEASURE_NUMBER_MODE, false, "measureNumberMode",     P_TYPE::INT     },
-
-      { P_ID::GLISS_TYPE,          false, 0,                       P_TYPE::INT     },
-      { P_ID::GLISS_TEXT,          false, 0,                       P_TYPE::STRING  },
-      { P_ID::GLISS_SHOW_TEXT,     false, 0,                       P_TYPE::BOOL    },
-      { P_ID::DIAGONAL,            false, 0,                       P_TYPE::BOOL      },
-      { P_ID::GROUPS,              false, 0,                       P_TYPE::GROUPS    },
-      { P_ID::LINE_STYLE,          false, "lineStyle",             P_TYPE::INT       },
-      { P_ID::LINE_COLOR,          false, 0,                       P_TYPE::COLOR     },
-      { P_ID::LINE_WIDTH,          false, 0,                       P_TYPE::SPATIUM   },
-      { P_ID::LASSO_POS,           false, 0,                       P_TYPE::POINT_MM  },
-      { P_ID::LASSO_SIZE,          false, 0,                       P_TYPE::SIZE_MM   },
-
-      { P_ID::TIME_STRETCH,        false, "timeStretch",           P_TYPE::REAL      },
```

Comment:
Be sure not to overlook this!  It appears you inadvertently changed stuff that probably breaks things here.  There was no reason any of this code should have shown as a diff.